convert2rhel/pkghandler.py

```diff
@@ -356,6 +357,11 @@ def replace_non_red_hat_packages():
 def preserve_only_rhel_kernel():
     loggerinst = logging.getLogger(__name__)
```

Comment:
Please instantiate the logger outside of the function scope, you probably don't want it re-instantiate each time you call this function. Pls see the following log example, when you have logger outside
```
----------------------------- live log collection ------------------------------
| 10:43:49 | my_pkg | INFO | __init__.py | Logger configured
| 10:43:49 | my_pkg.log | INFO | log.py | Logger obtained
collected 1 item

test_logger.py::test_foo 
-------------------------------- live log call ---------------------------------
| 10:43:49 | my_pkg.log | INFO | log.py | Foo called
| 10:43:49 | my_pkg.log | INFO | log.py | Foo called
PASSED
```

Reply:
/cc @SpyTec

Reply:
Outside the scope for this and #60, should be fixed in a separate PR.

As you [linked in #60](https://github.com/oamg/convert2rhel/pull/60#discussion_r432329984), Logging Cookbook mentions 

> Multiple calls to logging.getLogger('someLogger') return a reference to the same logger object.

If the method call returns a singleton I'm not sure I understand the reason for instantiating it outside method scope

Reply:
@SpyTec because it is an additional overhead which still returns the same object

Reply:
See https://github.com/oamg/convert2rhel/pull/60#discussion_r432385516. I agree with changing the way we work with the logger now. But let's do that under a separate PR for all the code.